package in.com.exception;

public class Testexceptoin {
public static void main(String[] args) {
	int a = 10;
	int b = 0;
	
	
	try {
		  int c = a/b;
	} catch (Exception e) {
		System.out.println(e);
	}
	
	
}
}
